devtools/maven/src/main/java/io/quarkus/maven/CreateProjectMojo.java

```diff
@@ -202,7 +218,18 @@ public void execute() throws MojoExecutionException {
             }
 
             success = createProject.execute().isSuccess();
-
+            if (success && parentPomModel != null) {
```

Comment:
We could also achieve that part directly in the maven codestart.. @aloubyansky what do you reckon?

Reply:
Probably, yes. This could be accepted until the codestarts provide this feature.

Reply:
https://github.com/quarkusio/quarkus/issues/12936